packages/cli/src/tests/transforms/transforms.test.ts

```diff
@@ -36,18 +36,18 @@ const exportValues = {
 };
 
 const expectedExports = {
-  commonJs: {
+  commonJs: expect.objectContaining({
     ...exportValues.commonJs,
     dyn: exportValues.all,
     default: {
       ...exportValues.commonJs,
       dyn: exportValues.all,
     },
-  },
-  module: {
+  }),
+  module: expect.objectContaining({
     ...exportValues.all,
     dyn: exportValues.all,
-  },
+  }),
```

Comment:
Turns out v24 was just returning some extra exports, but that doesn't break any of our regular usage patterns